content/en/docs/instrumentation/js/context.md

```diff
@@ -13,7 +13,7 @@ This document describes the OpenTelemetry context API for JavaScript and how it
 More information:
 
 - [Context specification](/docs/reference/specification/context/)
-- [Context API reference](https://open-telemetry.github.io/opentelemetry-js-api/classes/contextapi.html)
+- [Context API reference](https://open-telemetry.github.io/opentelemetry-js/classes/_opentelemetry_api.ContextAPI.html)
```

Comment:
@open-telemetry/javascript-approvers is this a valid updated link? If not, what should it be?

Reply:
If we don't hear back soon-ish then I'd so let's go with this, it's correct insofar as I'm not aware of any other context API reference